sumpy/visualization.py

```diff
@@ -163,9 +163,14 @@ def set_matplotlib_limits(self):
         pt.ylim((a[1], b[1]))
 
     def show_vector_in_mayavi(self, fld, do_show=True, **kwargs):
-        c = self.points
+        try:
+            from mayavi import mlab
+        except ImportError:
+            from warnings import warn
+            warn("mayavi is not available")
+            return
```

Comment:
I kind of liked it raising an error.

Reply:
Fair enough. I mostly did it like this to also shut pylint up :\ 

Reverted in
https://github.com/inducer/sumpy/compare/6be06e736169d8ea8f85fcef3c16a31b5d793415..e5ed7b5666c6b6b5ab3cfe838ac1f004bec4ef35